scripts/databases/auth.sh

```diff
@@ -8,6 +8,6 @@ exp=$(echo "$ACCESS_TOKEN" | jq -R 'split(".")? | .[1] | @base64d | fromjson | .
 NOW=$(date +%s)
 if [[ -z "$exp" ]] || (( $exp < $NOW )); then
   # shellcheck disable=SC2155
-  export ACCESS_TOKEN=$($HTTP --auth "$USER_EMAIL:$PASSWORD" post "$IM_HOST/tokens" | jq -r '.access_token')
+  export ACCESS_TOKEN=$($HTTP --auth "$USER_EMAIL:$PASSWORD" post "$IM_HOST/tokens" | jq -r '.accessToken')
```

Comment:
Why? :smile:

Reply:
Because the `Token`  field is also updated - https://github.com/dhis2-sre/im-manager/pull/276/files#diff-73f556a7b2309746d2b616a31bf69f24de0334f4707ce826f3ca185feadbef24R49-R52

Reply:
As mentioned on Slack... The reason for it being of that format is that I wanted to mimic the spring security implementation.

https://docs.spring.io/spring-security/oauth/apidocs/org/springframework/security/oauth2/common/OAuth2AccessToken.html

The motivation were spring being widely used and therefor a lot of 3rd party integrations would work out of the box. But I guess we don't need to integrate with many random 3rd party applications.

Another thing worth keeping in mind. This update will break interaction im-web-client. So that needs to be updated as well.